gocourse05/zooobservation/animal/animal.go

```diff
@@ -0,0 +1,16 @@
+package animal
+
+import (
+	"zooobservation/camera"
+	"zooobservation/common"
+)
+
+type Animal struct {
+	Id      int
```

Comment:
https://go.dev/wiki/CodeReviewComments#initialisms
```suggestion
	ID      int
```